### PR TITLE
Create stuff

### PR DESCRIPTION
**Pull request by coffeepac**
Create artifacts that will deploy kafka-monitor to a cluster that is running kafka so kafka can be monitored.

### CODE REVIEW

**Comment by davidewatson** [comment]
Per our conversation, we should have an issue to add Prometheus integration (https://prometheus.io/docs/instrumenting/exporters/). Also, since all the container does is install a configuration file, I think you should create a config map and submit this to k2-charts instead.

File: kafka-monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
```
Comment (davidewatson) [inline kafka-monitor.yaml]: This should be a resource controller deployment to allow ensure it is always running and upgradable.

File: kafka-monitor.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      imagePullPolicy: Always
+      ports:
+      - containerPort: 8000
+        name: gui
+      - containerPort: 8778
+        name: metrics
+
+---
+apiVersion: v1
+kind: Service
```
Comment (davidewatson) [inline kafka-monitor.yaml]: Each resource should be in a separate file.

**Comment by coffeepac** [comment]
@davidewatson there is now a ticket for prometheus exporters:  https://hanjin.atlassian.net/browse/ADA-33

File: kafka-monitor.yaml (hunk 2 above)
Comment (coffeepac) [inline kafka-monitor.yaml]: Why a separate file for each?  I kind of like the convenience of 'one stop shopping' for all components of this system.

Comment (davidewatson) [inline kafka-monitor.yaml]: You're right, according to the kubernetes best practices documentation (http://kubernetes.io/docs/user-guide/config-best-practices/), you should "group related objects together in a single file where this makes sense." kraken-services does this. k2-charts does not.

File: kafka-monitor.yaml (hunk 1 above)
Comment (coffeepac) [inline kafka-monitor.yaml]: I've been working on making this a deployment instead of a Pod.  Spinning up a cluster to test it out now.

**Comment by coffeepac** [comment]
@davidewatson yaml is a deployment now, not a pod.  should be good to go!

**Pull request merged**